plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/util/ParquetUtil.java

```diff
@@ -30,10 +33,7 @@
 import org.apache.iceberg.util.BinaryUtil;
 import org.apache.iceberg.util.UnicodeUtil;
 import org.apache.parquet.column.statistics.Statistics;
-import org.apache.parquet.hadoop.metadata.BlockMetaData;
-import org.apache.parquet.hadoop.metadata.ColumnChunkMetaData;
 import org.apache.parquet.hadoop.metadata.ColumnPath;
```

Comment:
Although it has `hadoop` in the name `ColumnPath` lives in `parquet-common` not `parquet-hadoop`

Reply:
Wow, nice job. And it looks like that class is only used in `parquet-hadoop`, nowhere in `parquet-common`.